scoreboard-colors.scss

```diff
@@ -399,3 +293,11 @@ $color-tree-editable-highlight: $color-gold;
     }
   }
 }
+
+// Expose the all the palette colors as css3 variables. Used by JS components
+:root {
+  --palette-colors: #{$palette-colors};
```

Comment:
I don't think I've seen this done before. Not entirely sure how it works. It stores the entire list as a variable?

Reply:
Yea, it stores it as key/values which I then read in JS. 
<img width="490" alt="Screenshot 2023-02-28 at 2 18 30 PM" src="https://user-images.githubusercontent.com/203161/221956582-9f4878e4-175f-44e1-8acb-92028d91d4a1.png">

Reply:
Ah, the output was different than I thought it would be. Thanks for the explanation!